### PR TITLE
Add logging back to pubspec

### PR DESCRIPTION
**Pull request by danielharasymiw-wf**
### Story:
TODO

### Acceptance Criteria:
- [ ] Code has been tested and results documented
- [ ] Unit tests have been updated
- [ ] Updates to documentation if necessary
- [ ] Verify and document changes to any other Messaging components
- [ ] Pull request made against the 'develop' branch, not master

### Design Notes:
TODO

### How To Test:
TODO

### My Test Results:
TODO

#### Reviewers:
@Workiva/service-platform

### CODE REVIEW

**Review by corwinsheahan-wf — APPROVED** [review]

**Comment by aviary-wf** [comment]
## Security Insights
No security relevant content was detected by automated scans.

## Action Items
* Review PR for [security impact](https://wiki.atl.workiva.net/display/SECURITY/Development+Security+Review+Guidelines); comment "security review required" if needed or unsure
* Verify [`aviary.yaml`](https://wiki.atl.workiva.net/pages/viewpage.action?pageId=42762752#SecurityInsightFramework&Tooling-ConfiguringAviary) coverage of security relevant code
---
_Questions or Comments? Reach out on Slack: #support-infosec._

**Comment by corwinsheahan-wf** [comment]
QA +1
* Non-functional change, CI Passing is sufficient.

File: lib/dart/pubspec.yaml [hunk 1]
```diff
@@ -1,4 +1,5 @@
 dependencies:
+  logging: ^1.0.0
```
Comment (brettkail-wk) [inline lib/dart/pubspec.yaml:2]: It seems like [this line](https://github.com/Workiva/frugal/blob/27e22767ff612ea81b1740d636ca7ad51d084de2/lib/dart/lib/src/frugal.dart#L22) should be deleted instead?

Comment (danielharasymiw-wf) [reply lib/dart/pubspec.yaml:2]: It looks to me like theres still logging usages https://github.com/Workiva/frugal/blob/master/lib/dart/lib/src/frugal/transport/f_adapter_transport.dart#L21

**Comment by danielharasymiw-wf** [comment]
@Workiva/release-management-p

**Review by rmconsole-wf — APPROVED** [review]
+1 from RM

**Pull request merged**